plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.containers;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Closer;
+import org.testcontainers.containers.Network;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.containers.Network.newNetwork;
+
+public class HiveMinioDataLake
+        implements Closeable, AutoCloseable
+{
+    public static final String ACCESS_KEY = "accesskey";
+    public static final String SECRET_KEY = "secretkey";
+
+    private final String bucketName;
+    private final Minio minio;
+    private final HiveHadoop hiveHadoop;
+
+    private final Closer closer = Closer.create();
+
+    public HiveMinioDataLake(String bucketName, Map<String, String> hiveHadoopFilesToMount)
+    {
+        this(bucketName, hiveHadoopFilesToMount, HiveHadoop.DEFAULT_IMAGE);
```

Comment:
nit: static import.

Reply:
Please do not static import that. DEFAULT_IMAGE itself does not tell much.